test/fork/Fork.t.sol

```diff
@@ -459,4 +469,24 @@ contract ForkTest is Test {
             abi.decode(deployments[id].parseRaw(".isTestnet"), (bool))
         );
     }
+
+    function testIntegrations() public {
+        if (vm.envOr("FORK_TESTS", false)) {
+            for (uint256 i = 0; i < deployments.length; i++) {
+                address cfg = _get(i, ".integrations.cfg");
+                address verUSDC = _get(i, ".integrations.verUSDC");
+                address root = _get(i, ".contracts.root");
+                address admin = _get(i, ".config.admin");
+                _loadFork(i);
+
+                if (verUSDC != address(32)) {
+                    assertTrue(IAuth(verUSDC).wards(root) == 1);
+                    assertTrue(IAuth(IWrappedUSDC(verUSDC).memberlist()).wards(admin) == 1);
+                }
+                if (cfg != address(32)) {
+                    assertTrue(IAuth(cfg).wards(root) == 1);
```

Comment:
Actually some other easy sanity checks we can add:
- `name = Centrifuge`
- `symbol = CFG` except on Ethereum where it's `wCFG`
- `decimals = 18`